deploy/vulnerability/README.md

```diff
@@ -6,7 +6,7 @@ We will do that by running cloudbeat with a custom `cloudbeat.yml` file that tri
 
 ### How to run it
 *Prerequisites:*
-1. You have an elasticsearch deployed locally or in the cloud
+1. You have an elasticsearch deployed locally or in the cloud (check https://github.com/elastic/elastic-package to run elastic stack locally)
```

Comment:
I think we can ref our docs here https://github.com/elastic/cloudbeat/blob/main/dev-docs/ELK-Deployment.md